clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/controllers/servergroup/ServerGroupEventStatusJudge.java

```diff
@@ -0,0 +1,27 @@
+package com.netflix.spinnaker.clouddriver.ecs.controllers.servergroup;
+
+import com.amazonaws.services.ecs.model.ServiceEvent;
+import com.netflix.spinnaker.clouddriver.ecs.model.EcsServerGroupEventStatus;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ServerGroupEventStatusJudge {
```

Comment:
The name of this class doesn't really reflect the behavior of the class; it's not really judging anything so much as converting an event to a status. Really should be `ServerGroupEventStatusConverter` or similar.

Reply:
I'd also maybe consider just putting this logic onto `EcsServerGroupEventStatus` as a static method. For example, you could do `EcsServerGroupEventStatus.fromServiceEvent(event)` instead. 

Up to you on which way to go.

Reply:
I've renamed it to `ServerGroupEventStatusConverter` as suggested.  I'd rather keep that conversion logic in that separate class as it's likely it will need to evolve together with the ECS API evolving, so I'd like to keep that piece away from the ServerGroupController